Configuration/DataProcessing/python/Impl/ppEra_Run2_2017_trackingOnly.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+"""
+_ppEra_Run2_2017_trackingOnly
+
+Scenario supporting proton collisions and tracking only reconstruction for HP beamspot
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Reco import Reco
+import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+
+from Configuration.DataProcessing.Impl.pp import pp
+
+class ppEra_Run2_2017_trackingOnly(pp):
```

Comment:
please follow the same pattern as in hcalnzs:
define a default scenario first and then add the era-based modifiers on top.
This way reduces maintenance requirements for the future.